web/res/js/membershipForm.js

```diff
@@ -150,12 +150,8 @@ $( function () {
 				stateKey: 'membershipFormContent.membershipType'
 			},
 			{
-				viewHandler: WMDE.View.createFeeOptionSwitcher( $( '#amount-1' ), 1 ),
-				stateKey: 'membershipFormContent.paymentIntervalInMonths'
-			},
-			{
-				viewHandler: WMDE.View.createFeeOptionSwitcher( $( '#amount-2' ), 6 ),
-				stateKey: 'membershipFormContent.paymentIntervalInMonths'
+				viewHandler: WMDE.View.createFeeOptionSwitcher( [ $( '#amount-1' ), $( '#amount-2' ), $( '#amount-3' ), $( '#amount-4' ), $( '#amount-5' ), $( '#amount-6' ), $( '#amount-7' ) ], { person: 24, firma: 100 } ),
+				stateKey: 'membershipFormContent'
```

Comment:
These limits are hardcoded as a constant in `MembershipFeeValidator`. In some cases it would be nice to let client-side and server-side code share some configuration. Also regarding the messages (we keep them in JSON anyway). But we have to be careful, since there is stuff in the config we wouldn't want the client know about.